### PR TITLE
VITIS-9224 Xbutil Enhancements for Power Management

### PR DESCRIPTION
**Pull request by AShivangi**
#### Problem solved by the commit
Plumbing for
- [VITIS-9561](https://jira.xilinx.com/browse/VITIS-9561) Set high, medium and low performance mode
- [VITIS-9562](https://jira.xilinx.com/browse/VITIS-9562) Report current frequencies and performance mode

#### Bug / issue (if any) fixed, which PR introduced the bug, how it was discovered
N/A
#### How problem was solved, alternative solutions (if any) and why they were rejected
Created a new subcommand to set performance and display which mode it is in platform report

#### Risks (if any) associated the changes in the commit
None

#### What has been tested and how, request additional testing if necessary
Tested with simulated values:
```
$ xbutil examine -d b3:00 -r platform

---------------------------------------------------
[0000:b3:00.1] : xilinx_vck5000_gen4x8_qdma_base_2
---------------------------------------------------
Platform
  XSA Name               : xilinx_vck5000_gen4x8_qdma_base_2 
  Logic UUID             : 05DCA096-76CB-730B-8D19-EC1192FBAE3F 
  FPGA Name              :  
  JTAG ID Code           : 0x0 
  DDR Size               : 0 Bytes
  DDR Count              : 0 
  Revision               : 1 
  MFG Date               : 0xcee875 
  Mig Calibrated         : false 
  P2P Status             : not supported 
  Performance Mode       : Default

Clocks
  DATA_CLK (Data)        : 300 MHz
  KERNEL_CLK (Kernel)    : 500 MHz

Mac Addresses            : 00:0A:35:0D:55:8E
                         : 00:0A:35:0D:55:8F
```
```
$ xbutil configure --performance -d b3:00 low

Performance mode is set to low 

```

Negative test:
```
$ xbutil configure --performance -d b3:00 lo
ERROR: Invalid action value: 'lo'

COMMAND: configure

DESCRIPTION: Change performance mode of the device

USAGE:  configure --performance [--help] [-d arg] action

OPTIONS:
  --performance      - Change performance mode of the device
  -d, --device       - The Bus:Device.Function (e.g., 0000:d8:00.0) device of interest
  --help             - Help to use this sub-command


ERROR: : Operation canceled
```

#### Documentation impact (if any)
Added the new command usage to github.io

### CODE REVIEW

File: src/runtime_src/doc/toc/xbutil.rst [hunk 1]
```diff
@@ -226,6 +226,12 @@ Command ``xbutil configure`` is used to configure specific settings based on the
 - The ``--size`` is used in conjuction with ``xbutil configure --host-mem enable`` to specify the host-memory size to be enabled
     
     - ``<size>``: Size and unit specified as a combined string 
+- The ``--performance`` select specific configuration for benchmarking tests
+    
+    - ``low``: Set performance mode to low
+    - ``medium``: Set performance mode to medium
+    - ``medium``: Set performance mode to high
```
Comment (uday610) [inline src/runtime_src/doc/toc/xbutil.rst:233]: ``high`` :

File: src/runtime_src/doc/toc/xbutil.rst [hunk 2]
```diff
@@ -226,6 +226,12 @@ Command ``xbutil configure`` is used to configure specific settings based on the
...
+    - ``medium``: Set performance mode to default
```
Comment (uday610) [inline src/runtime_src/doc/toc/xbutil.rst:234]: default

**Review by uday610 — CHANGES_REQUESTED** [review]

**Review by chvamshi-xilinx — APPROVED** [review]
Looks good.

File: src/runtime_src/doc/toc/xbutil.rst (hunk 1 above)
Comment (AShivangi) [reply src/runtime_src/doc/toc/xbutil.rst]: fixed

File: src/runtime_src/doc/toc/xbutil.rst (hunk 2 above)
Comment (AShivangi) [reply src/runtime_src/doc/toc/xbutil.rst]: fixed

**Pull request merged**